packages/next/src/lib/turbopack-warning.ts

```diff
@@ -205,36 +203,32 @@ export async function validateTurboNextConfig({
       `\n${chalk.yellow(
         'Warning:'
       )} Webpack is configured while Turbopack is not, which may cause problems.\n
-  ${`See instructions if you need to configure Turbopack:\n  https://turbo.build/pack/docs/features/customizing-turbopack\n`}`
+See instructions if you need to configure Turbopack:\n  https://turbo.build/pack/docs/features/customizing-turbopack\n`
     )
   }
 
   if (babelrc) {
     unsupportedParts += `\n- Babel detected (${chalk.cyan(
       babelrc
-    )})\n  ${`Babel is not yet supported. To use Turbopack at the moment,\n  you'll need to remove your usage of Babel.`}`
+    )})\n  Babel is not yet supported. To use Turbopack at the moment,\n  you'll need to remove your usage of Babel.`
   }
   if (unsupportedConfig.length) {
     unsupportedParts += `\n\n- Unsupported Next.js configuration option(s) (${chalk.cyan(
       'next.config.js'
-    )})\n  ${`To use Turbopack, remove the following configuration options:\n${unsupportedConfig
+    )})\n  To use Turbopack, remove the following configuration options:\n${unsupportedConfig
       .map((name) => `    - ${chalk.red(name)}\n`)
-      .join('')}  `}   `
+      .join('')}`
   }
 
   if (unsupportedParts && !isCustomTurbopack) {
     const pkgManager = getPkgManager(dir)
 
     console.error(
-      `${'Error:'} You are using configuration and/or tools that are not yet\nsupported by Next.js v13 with Turbopack:\n${unsupportedParts}\n
+      `Error: You are using configuration and/or tools that are not yet\nsupported by Next.js v13 with Turbopack:\n${unsupportedParts}\n
 If you cannot make the changes above, but still want to try out\nNext.js v13 with Turbopack, create the Next.js v13 playground app\nby running the following commands:
 
   ${chalk.bold.cyan(
-    `${
-      pkgManager === 'npm'
-        ? 'npx create-next-app'
-        : `${pkgManager} create next-app`
-    } --example with-turbopack with-turbopack-app`
+    `${pkgManager} create next-app --example with-turbopack with-turbopack-app`
```

Comment:
Can we revert the changes in this file to keep it isolated to the intended fix? 🙏